app.js

```diff
@@ -14,18 +14,44 @@ Words only consist of letters, never apostrophes or other punctuation symbols.
   - identify most frequent word and return 
   */
 
- var nDictionary = Object.create(null);
+ function getWordsWithHighestFreq(paragraphArr) { 
+    var frequencyDict = new Object(); 
 
- function setDictionary(index, value) {
-     nDictionary[index] = value;
- }
- 
- function getDictionary(index) {
-     return nDictionary[index];
- }
+    //Adds words from paragraph into a dictionary 
+    for (var i = 0; i < paragraphArr.length; i++) {
+      var key = paragraphArr[i]; 
 
- getDictWithFrequencies(paragraphArr, bannedWordsArr) { 
-    var frequencyDict = new Object(); 
+      if(key in frequencyDict) {
```

Comment:
I'd add a space between 'if' and first ( 
Same for 'for' loops (line 21 looks good)

Reply:
Also, haven't seen (propertyName in objectName) used before--that's cool. Most of the time I tend to see objectName.hasOwnProperty(propName), which makes sure that the property is not on the object's prototype